sympy/simplify/trigsimp.py

```diff
@@ -438,6 +438,18 @@ def check_args(x, y):
     def f(rv):
         # for simple functions
         g = getattr(rv, 'inverse', None)
+        if isinstance(rv, asin):
+            x = rv.args[0]
+            if _coeff_isneg(x):
+                print(x.args)
+                return -f(x.args[1].args[0])
```

Comment:
This does not check whether the argument is a sin function.

Also print should not be used.

Reply:
Thanks, I'll fix it when I'm back